contracts/src/Teleporter/ReceiptQueue.sol

```diff
@@ -7,15 +7,19 @@ pragma solidity 0.8.18;
 
 import "./ITeleporterMessenger.sol";
 
+// The maximum number of receipts to include in a single message.
+uint256 constant MAXIMUM_RECEIPT_COUNT = 5;
```

Comment:
any reason we define the constant outside library?

Reply:
moved into library.